app/src/main/java/com/styleru/muro/androidmsk/Network/Network.kt

```diff
@@ -0,0 +1,45 @@
+package com.styleru.muro.androidmsk.Network
+
+import com.styleru.muro.androidmsk.BASE_URL
+import io.reactivex.schedulers.Schedulers
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Network private constructor() {
```

Comment:
В котлин встроена поддержка Singleton
```suggestion
object Network {
```